source/_integrations/switchbot_cloud.markdown

```diff
@@ -0,0 +1,39 @@
+---
+title: SwitchBot Cloud
+description: Instructions on how to set up SwitchBot Devices.
+ha_category:
+  - Switch
+ha_release: '2023.10'
+ha_iot_class: Cloud Polling
+ha_codeowners:
+  - '@SeraphicRav'
+ha_domain: switchbot_cloud
+ha_bluetooth: false
+ha_platforms:
+  - switch
+ha_config_flow: true
+ha_integration_type: integration
+---
+
+The SwitchBot integration allows you to control SwitchBot [devices](https://www.switch-bot.com/).
+
+## Prerequisites
+
+In order to use this integration, you will need to get a token and secret key from the SwitchBot mobile app in Profiles>Preferences>Developer Options.
+
+Please note, device names configured in the SwitchBot app are transferred into Home Assistant.
+
+{% include integrations/config_flow.md %}
+
+## Supported Devices
+
+- Plug (Wi-Fi only), Plug Mini, both the original (model W1901400) and HomeKit-enabled (model W1901401)
```

Comment:
This is a strange sentence, is "Plug (Wi-Fi only)" 1 product and is Plug Mini another one? And with the text behind it, are there 4 products?

(when you own one you probably know what this means, but if you're out there looking in the docs if something is supported this can get confusing I think)

Reply:
Confusing right ? There are actually 5 models:
- Plug (SP11, only Wi-FI and only available in Japan)
- Plug Mini which exist both in US and JP model with a variant with and without HomeKit support

Should I explain it this way ?

Reply:
hahaha what

To maybe make it more relatable to the product, how does Switchbot show this on their website?

If we support all Plug Mini's we could just say that we support Plug Mini, no need to overload (this works until they bring out another variant).

Reply:
@joostlek I think I simplified the explanation, please tell me if it is better

Reply:
I think is clear. I am not an expert on these devices, but I think I would be able to tell if mine was supported with this explanation

Reply:
For the website, in Japan, they have 3 different products:
- Plug : https://www.switchbot.jp/products/switchbot-plug?variant=40981729968303
- Smart Plug Mini with works with Homekit : https://www.switchbot.jp/pages/switchbot-plug-mini-homekit
- Smart Plug Mini : https://www.switchbot.jp/pages/switchbot-plug-mini